integrations/symfony-bundle.rst

```diff
@@ -16,7 +16,10 @@ Install the HTTPlug bundle with composer and enable it in your AppKernel.php.
 
 .. code-block:: bash
 
-    $ composer require php-http/httplug-bundle
+    $ composer require php-http/httplug-bundle <your client adapter choice>
+
+If you did not install HTTPlug itself, you will also need to specify which HTTP
```

Comment:
I do not understand this line. Are you trying to say something like:
"If composer fails to install the bundle, you have to specify which HTTP..."